qiita_db/test/test_analysis.py

```diff
@@ -95,12 +100,12 @@ def test_create(self):
 
         new = Analysis.create(User("admin@foo.bar"), "newAnalysis",
                               "A New Analysis")
-        self.assertEqual(new.id, 3)
+        self.assertEqual(new.id, 7)
```

Comment:
Just for completeness all the SQL queries/tests on this function will be parametrized.